src/nvim/lua/treesitter.c

```diff
@@ -1073,7 +1073,10 @@ static int node_rawquery(lua_State *L)
   // TODO(bfredl): these are expensive allegedly,
   // use a reuse list later on?
   TSQueryCursor *cursor = ts_query_cursor_new();
+  // TODO(clason): use of prerelease API; remove guard after ts bump to 0.19.6
```

Comment:
No, we'll still need to keep it if people build against 0.19.5 or earlier.  I don't see why we need to bump our minimum required version to 0.19.6 when it's released.

Reply:
Not need, but probably want. I assume that we will before long bump the minimum required version to 0.19.6 before long; this reminds me to remove the guard only _if_ the bump happens.

But I can also remove the TODO if you prefer?

Reply:
lint wasn't very happy with that TODO, either 😆

Reply:
The TODO is fine, but the comment implies it's _only_ here due to building against a pre-release, which isn't true.

Reply:
Or I could reformulate "remove guard after minimum version is bumped to 0.19.6+"? That's a clearer statement of the intention.

Reply:
well it is quite likely we will depend on yet newer TS API during the 0.6 cycle. While there is no hurry to remove it right after 0.19.6 is released, it is probably going to be defunct not long after.

Reply:
stupid MSVC build failed anyway, might as well push an update 😄